tests/ExcelFinancialFunctions.ConsoleTests/MatrixTests.fs

```diff
@@ -0,0 +1,133 @@
+﻿namespace Excel.FinancialFunctions
```

Comment:
Were some tests deleted?  I just see additions, which is cool, just wondering

Reply:
No tests removed or deleted or even modified. At this point I probably don't have the F# skills to be making changes like that 😃

Just reorganized HOW they were run. I did remove the file "testsexec.fs" from the project. Left the file itself in the repository for you to refer to if you wanted. All the work from testsexec.fs has been moved into SpotTests.fs and MatrixTests.fs.

Reply:
I see, yes just remove testexec.fs then?

Reply:
Will do. Was going to come back and do that as a cleanup